lib/src/pubspec.dart

```diff
@@ -105,6 +106,6 @@ class Pubspec {
     if (vc == null) {
       return false;
```

Comment:
so if there is no constraint on the sdk we don't warn? Ah, because that's a different problem: `hasDartSdkConstraint`

Reply:
Yeah, `hasDartSdkConstraint` handles that part.